source/stylesheets/refills/unstyled/_navigation.scss

```diff
@@ -0,0 +1,82 @@
+header.navigation {
+  $large-screen: em(860) !default;
+  $large-screen: $large-screen;
+
+  // Mobile view
+  .navigation-menu-button {
+    display: block;
+    float: right;
+    margin: 0;
+    padding-top: 0.5em;
+
+    @include media ($large-screen) {
+      display: none;
+    }
+  }
+
+  // Nav menu
+  .navigation-wrapper {
+    @include clearfix;
+    position: relative;
+  }
+
+  .logo {
+    float: left;
+
+    img {
+      max-height: 2em;
+      padding-right: 1em;
+    }
+  }
+
+  nav {
+    float: none;
+
+    @include media ($large-screen) {
+      float: left;
+      line-height: 1.5em;
+      padding-top: 0.3em;
+    }
+  }
+
+  ul.navigation-menu {
```

Comment:
The nav is a fairly tricky pattern with a lot of scanning back and forth between html and scss. I really think it helps adding this extra bit of information in scss. I'd like to keep it.